app/src/main/java/com/stl/skipthelibrary/MyBooksActivity.java

```diff
@@ -11,19 +15,34 @@
 import androidx.recyclerview.widget.RecyclerView;
 
 public class MyBooksActivity extends AppCompatActivity {
+    private static final String TAG = MyBooksActivity.class.getSimpleName();
+    public static final int ADD = 1;
+
     ArrayList<Book> books = new ArrayList<Book>();
     RecyclerView recyclerView;
+    FloatingActionButton addBookButton;
+    Context mContext;
```

Comment:
These should probably be private as well

Reply:
If not please specify their access (e.g. public or protected)

Reply:
This is also my fault as I forgot to specify the access of books and recycler view